esmvaltool/cmorizers/data/formatters/datasets/noaa_mbl_ch4.py

```diff
@@ -0,0 +1,164 @@
+"""ESMValTool CMORizer for NOAA MBL CH4 data.
+
+Tier
+    Tier 2: other freely-available dataset.
+
+Source
+    https://gml.noaa.gov/ccgg/trends_ch4/
+
+Last access
+    20230717
+
+Download and processing instructions
+    Download the file:
+    wget https://gml.noaa.gov/webdata/ccgg/trends/ch4/ch4_mm_gl.csv
+    or
+    use the automatic download script shipped by ESMValTool
+    esmvaltool data download NOAA-MBL-CH4
+    or
+    esmvaltool data prepare NOAA-MBL-CH4
```

Comment:
I would recommend to remove these lines from the header. It is usually only needed to document ways to manually download data. The esmvaltool download commands apply to many datasets and are already described in the documentation.